cookowly-frontend/playwright.config.ts

```
@@ -1,17 +1,19 @@
 import type { PlaywrightTestConfig } from '@playwright/test';
 import { devices } from '@playwright/test';
+import * as dotenv from 'dotenv';
+// see https://github.com/motdotla/dotenv#how-do-i-use-dotenv-with-import
+dotenv.config();
 
-/**
- * Read environment variables from file.
- * https://github.com/motdotla/dotenv
- */
-// require('dotenv').config();
+const host = process.env.PW_BASE_URL || 'http://localhost';
+const port = process.env.PW_APP_PORT || 3000;
+const baseURL = `${host}:${port}`;
```

we could maybe let it crash to show potential env setup errors

---

what do you mean?
something like a new line with 
If (!process.env....) { throw new Error('...') }

?

---

this or just remove check and let playwright crash later on

---

Playwright will not "crash" by itself
It will tell you, that the browser could not visit the page but that's it.
Will build an explicit crash